src/components/pages/HomePage.js

```diff
@@ -0,0 +1,311 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Menu, Modal, Input, Grid, Button, Form } from 'semantic-ui-react';
+import { Link } from 'react-router-dom'
+
+import requestPipeline from '../../components/pages/requestPipeline';
+
```

Comment:
We should separate the imports by importing things we create and other packages we did not make. Why is this import on its own?